src/components/dialogs/filter/filters-save.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { createFilter, saveFilter } from '../../../utils/rest-api';
+import { FilterType } from '../../../utils/elementType';
+import { frontToBackTweak } from './criteria-based-filter-dialog-utils';
+import { NAME } from '../../utils/field-constants';
+
+export const saveExplicitNamingFilter = (
```

Comment:
I don't think we need this method, we can use createFilter  in create-filter-dialog.js and  saveFilter in explicit-naming-filter-edition-dialog.js directly

Reply:
I kept the previous code the more I can (it's what tech lead usually told me to do)

Reply:
I kept the previous code the more I can (it's what tech lead usually told me to do)

Reply:
I kept the previous code the more I can (it's what tech lead usually told me to do)